### DISCUSSION TITLE
是不是没有用到pysot/training_dataset/vid/val.jsonvid/val.json文件？

### DISCUSSION THREAD

**Discussion by haishibei** [STVIR/pysot#365] [General]
我想问一下，在训练SiamFC的时候，代码中是不是没有用到 pysot/training_dataset/vid/val.json文件？如果是这样，那么，是不是说训练的时候没有用到vid里面验证集（pysot/training_dataset/vid/crop511/e）?      (这样的话，验证集不就浪费了吗？)

**Comment by ZhiyuanChen** [comment]
emmm
为什么你会想在训练的时候加入验证集呢？

**Comment by haishibei** [comment]
> 
> 
> emmm
> 为什么你会想在训练的时候加入验证集呢？

感谢您的回复，可能我没有表达清楚，我没有想在训练的时候加验证集，我想问的是，数据预处理中，对ILSVRC2015的验证集val进行了剪裁（软连接e所对应的部分），但是pysot/training_dataset/vid/train.json并没有包含该部分(pysot/training_dataset/vid/crop511/e)，所以在训练的时候是不是没有用到这部分数据集？如果训练的时候没有用到，为什么还要对它进行预处理剪裁哪？

**Comment by ZhiyuanChen** [comment]
> 数据预处理中，对ILSVRC2015的验证集val进行了剪裁（软连接e所对应的部分），但是pysot/training_dataset/vid/train.json并没有包含该部分(pysot/training_dataset/vid/crop511/e)，所以在训练的时候是不是没有用到这部分数据集？如果训练的时候没有用到，为什么还要对它进行预处理剪裁哪？

这看上去有些像一个bug

**Comment by xxAna** [comment]
我见过有些追踪器的训练会包含验证集上的验证部分，是不是因为没写这部分代码所以没用到呢？
